terraform/modules/paas/variables.tf

```diff
@@ -19,6 +19,9 @@ variable web_app_memory { default = 512 }
 variable worker_app_instances { default = 1 }
 variable worker_app_memory { default = 512 }
 
+variable logging_endpoint_port {}
```

Comment:
can we call it `logit_url` or just `logging_url`?
also can this be moved to `locals` as we are going to just have one logit stack for all envs?

Reply:
`log_url` looks fine
We need to keep it as an input variable since this should be considered a secret

Reply:
done...